src/Client.php

```diff
@@ -120,6 +120,8 @@ public function sendAsync(RequestInterface $request, array $options = []): Promi
     public function send(RequestInterface $request, array $options = []): ResponseInterface
     {
         $options[RequestOptions::SYNCHRONOUS] = true;
+
+        /** @var ResponseInterface */
```

Comment:
All the `@var` tags added here is due to `guzzlehttp/promises`'s interface `PromiseInterface` having `mixed` as the return type